cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -168,7 +168,7 @@ resource {{bicepName $name}} 'Microsoft.App/containerApps@2023-05-02-preview' =
       containers: [
         {
           image: '{{$value.Image}}'
-          name: '{{$name}}'
+          name: '{{containerAppName $name}}-${resourceToken}'
```

Comment:
The `containerAppName` which is bound to `scaffold.ContainerAppName` was meant for `Microsoft.ContainerApps`, not for the container within -- naming restrictions may be different.

Reply:
What are the character restrictions here? FYI, recently Aspire has started restricting resource identifiers to alphanumeric + dash, with no dash as the first or last character.